test/finset_repr.lean

```diff
@@ -0,0 +1,11 @@
+import Mathlib.Data.Finset.Sort
+
+#eval show Lean.MetaM Unit from guard <|
+  (repr (0 : Multiset (List ℕ)) |>.pretty 15) = "0"
```

Comment:
These tests might be clearer with a simple `#guard_msgs`, rather than mentioning `MetaM`.

Reply:
I specifically want to use a narrow width, so would prefer to test equality.

Is there a better way to write simple assertions than what I do here?

Reply:
There are a handful of other tests in this style, so I'm going to leave it as is for now

bors merge